tests/unittests/OperatorTest.cpp

```diff
@@ -3362,6 +3362,63 @@ TEST_P(InterpAndCPU, insertTensorTest) {
   }
 }
 
+/// Test RowwiseQuantizedFullyConnected Node.
+TEST_P(InterpOnly, rowwiseQuantizedFCTest) {
+  // In this test we subtract the outputs of a row-wise quantized FC and a
+  // floating-point FC and ensure that the error is below some low value.
+  auto *input = mod_.createVariable(ElemKind::FloatTy, {1, 300}, "in");
+  auto *fc = F_->createFullyConnected("FC", input, 30);
+  auto *res =
+      mod_.createVariable(ElemKind::FloatTy, fc->getResult().dims(), "res");
+
+  auto weights = fc->getWeights();
+  auto bias = fc->getBias();
+
+  input->getPayload().getHandle().randomize(-1.0, 1.0, mod_.getPRNG());
+  llvm::cast<Variable>(bias)->getPayload().getHandle().randomize(
+      0, 0.00001, mod_.getPRNG());
+  llvm::cast<Variable>(weights)->getPayload().getHandle().randomize(
+      -1.1, 1.1, mod_.getPRNG());
+
+  // The FC fomula is I * W + B, while the RWQFC is I * transpose(W) + B.
+  // So get the tranpose of weights from the above FC.
+  auto *newWeights = mod_.createVariable(
+      ElemKind::FloatTy, {weights.dims()[1], weights.dims()[0]}, "newW");
+
+  newWeights->getPayload().transpose(
+      &(llvm::cast<Variable>(weights)->getPayload()), {1, 0});
+
+  TypeRef resTy = mod_.uniqueType(ElemKind::Int8QTy, res->dims(), 0.15, 4);
+  TypeRef biasTy = mod_.uniqueType(ElemKind::Int8QTy, bias.dims(), 0.02, 1);
+
+  TypeRef inputTy = mod_.uniqueType(ElemKind::Int8QTy, input->dims(), 0.01, 0);
+  auto *inputq = F_->createQuantize("input.q", input, inputTy);
+  auto *biasq = F_->createQuantize("bias.q", bias, biasTy);
+
+  auto *fcq = F_->createRowwiseQuantizedFullyConnected(
+      "fcq", inputq, newWeights, biasq, resTy);
+  auto *dequantRes = F_->createDequantize("dequant", fcq);
+
+  // Subtract the results of the convolution from the quantized fc.
+  auto *sub = F_->createSub("compare", dequantRes, fc);
+
+  F_->createSave("save", sub, res);
+  Context ctx;
+  EE_.compile(CompilationMode::Infer, F_, ctx);
+  EE_.run();
+
+  auto H = res->getPayload().getHandle();
+  // Check that there aren't too many elements with a difference in the results
+  // of greater than 0.2.
+  int count = 0;
+  for (int i = 0, e = H.size(); i < e; i++) {
+    if (std::abs(H.raw(i)) > 0.2) {
```

Comment:
why not here just do, EXPECT_LT(std::abs(H.raw(i)), 0.2)?

Reply:
it's hard to tell whether this works fine or not without knowing the actual numbers coming out of the FCs.
Have you looked at the results produced by rwFC and regular FC how big/small those are?